treq/auth.py

```diff
@@ -26,6 +182,112 @@ def request(self, method, uri, headers=None, bodyProducer=None):
             method, uri, headers=headers, bodyProducer=bodyProducer)
 
 
+class _RequestDigestAuthenticationAgent(object):
+
+    digest_auth_cache = {}
```

Comment:
When I put this inside of init it appeared to no longer cache the authentication across requests.